### PR TITLE
Workflow combine

### LINKED ISSUE #384: Upgrade the MyGet subscription
**Is your feature request related to a problem? Please describe.**
Workflow is failing on push, step publish nuget packages. Neo needs to upgrade the `MyGet` subscription to allow bigger packages. Or we need to change workflow.

```
Pushing Neo.Assertions.3.6.63-preview.nupkg to 'https://www.myget.org/F/neo/api/v2/package'...
  PUT https://www.myget.org/F/neo/api/v2/package/
  RequestEntityTooLarge https://www.myget.org/F/neo/api/v2/package/ 1[8](https://github.com/neo-project/neo-express/actions/runs/6989326929/job/19017796558#step:4:9)7ms
error: Response status code does not indicate success: 413 (The total packages size for the pending uploads is too large. The feed owner may have to upgrade the MyGet subscription to resolve this issue.).
```

**Describe the solution you'd like**
Upgrade subscription or change workflow to remove package upload or push to github

### PR DESCRIPTION
**Pull request by cschuchardt88**
@Liaojinghui this got overwritten when you merged development in master.

closes #384

### CODE REVIEW

**Review by Liaojinghui — APPROVED** [review]

**Comment by Liaojinghui** [comment]
The reason for this might be the pack action can not complete.

**Comment by cschuchardt88** [comment]
@Liaojinghui 
This step changes the `pack` is no longer required. You need to change that, so it not required and **ONLY** `test` is required.

PRs dont use `pack` anymore. Everything combined in `test` now.

**Comment by cschuchardt88** [comment]
@Liaojinghui me change something before merge, uploading to many packages at once. myget is mad about that.

**Comment by cschuchardt88** [comment]
@Liaojinghui ok good to go

File: .github/workflows/pack.yml [hunk 1]
```diff
@@ -38,21 +38,39 @@ jobs:
...
+      run: dotnet nuget push ./out/Neo.Collector.*.nupkg --source https://www.myget.org/F/neo/api/v3/index.json --api-key ${{ secrets.MYGET_TOKEN }}
+
+    - name: Publish Package to MyGet (Neo.Express)
+      run: dotnet nuget push ./out/Neo.Express.*.nupkg --source https://www.myget.org/F/neo/api/v3/index.json --api-key ${{ secrets.MYGET_TOKEN }}
+
+    - name: Publish Package to MyGet (Neo.Test.Harness)
+      run: dotnet nuget push ./out/Neo.Test.Harness.*.nupkg --source https://www.myget.org/F/neo/api/v3/index.json --api-key ${{ secrets.MYGET_TOKEN }}
+
+    - name: Publish Package to MyGet (Neo.Test.Runner)
+      run: dotnet nuget push ./out/Neo.Test.Runner.*.nupkg --source https://www.myget.org/F/neo/api/v3/index.json --api-key ${{ secrets.MYGET_TOKEN }}
```
Comment (Liaojinghui) [inline .github/workflows/pack.yml:69]:
```suggestion
      run: dotnet pack neo-express.sln --configuration ${{ env.CONFIGURATION }} --output ./out --no-build --verbosity normal

    - name: Publish Package to MyGet (Neo.Assertions)
      if: github.event_name == 'push' && (github.ref == 'refs/heads/master' || github.ref == 'refs/heads/development')
      run: dotnet nuget push ./out/Neo.Assertions.*.nupkg --source https://www.myget.org/F/neo/api/v3/index.json --api-key ${{ secrets.MYGET_TOKEN }}

    - name: Publish Package to MyGet (Neo.BlockchainToolkit)
      if: github.event_name == 'push' && (github.ref == 'refs/heads/master' || github.ref == 'refs/heads/development')
      run: dotnet nuget push ./out/Neo.BlockchainToolkit.*.nupkg --source https://www.myget.org/F/neo/api/v3/index.json --api-key ${{ secrets.MYGET_TOKEN }}

    - name: Publish Package to MyGet (Neo.BuildTasks)
      if: github.event_name == 'push' && (github.ref == 'refs/heads/master' || github.ref == 'refs/heads/development')
      run: dotnet nuget push ./out/Neo.BuildTasks.*.nupkg --source https://www.myget.org/F/neo/api/v3/index.json --api-key ${{ secrets.MYGET_TOKEN }}

    - name: Publish Package to MyGet (Neo.Collector)
      if: github.event_name == 'push' && (github.ref == 'refs/heads/master' || github.ref == 'refs/heads/development')
      run: dotnet nuget push ./out/Neo.Collector.*.nupkg --source https://www.myget.org/F/neo/api/v3/index.json --api-key ${{ secrets.MYGET_TOKEN }}

    - name: Publish Package to MyGet (Neo.Express)
      if: github.event_name == 'push' && (github.ref == 'refs/heads/master' || github.ref == 'refs/heads/development')
      run: dotnet nuget push ./out/Neo.Express.*.nupkg --source https://www.myget.org/F/neo/api/v3/index.json --api-key ${{ secrets.MYGET_TOKEN }}

    - name: Publish Package to MyGet (Neo.Test.Harness)
      if: github.event_name == 'push' && (github.ref == 'refs/heads/master' || github.ref == 'refs/heads/development')
      run: dotnet nuget push ./out/Neo.Test.Harness.*.nupkg --source https://www.myget.org/F/neo/api/v3/index.json --api-key ${{ secrets.MYGET_TOKEN }}

    - name: Publish Package to MyGet (Neo.Test.Runner)
      if: github.event_name == 'push' && (github.ref == 'refs/heads/master' || github.ref == 'refs/heads/development')
      run: dotnet nuget push ./out/Neo.Test.Runner.*.nupkg --source https://www.myget.org/F/neo/api/v3/index.json --api-key ${{ secrets.MYGET_TOKEN }}
```

File: .github/workflows/pr.yml [hunk 2]
```diff
@@ -12,10 +13,5 @@ on:
       - '**/*.md'
 
 jobs:
-  pack:
-    uses: ./.github/workflows/pack.yml
   test:
     uses: ./.github/workflows/test.yml
```
Comment (Liaojinghui) [inline .github/workflows/pr.yml:17]:
```suggestion
  test:
    uses: ./.github/workflows/test.yml
  pack:
    name: Test Pack
    needs: [test]
    uses: ./.github/workflows/pack.yml
```

File: .github/workflows/pr.yml [hunk 3]
```diff
@@ -12,10 +13,9 @@ on:
...
-  neoxp:
-    needs: [pack, test]  
-    uses: ./.github/workflows/neoxp.yml
+  pack:
+    name: Test Pack
```
Comment (Liaojinghui) [inline .github/workflows/pr.yml:19]:
```suggestion
    name: Pack
```

**Pull request merged**